builder/builder.go

```diff
@@ -364,14 +364,18 @@ func (b *Builder) BuildUpdate(params UpdateParameters) error {
 		}
 	}
 
-	// Publish. Update the latest version both in the format (used by update itself) and in LAST_VER
-	// (used to create newer manifests).
+	// Publish. Update the latest version file in various locations.
 	if !params.Publish {
 		return nil
 	}
 
 	fmt.Printf("Setting latest version to %s\n", b.MixVer)
 
+	err = ioutil.WriteFile(filepath.Join(b.Config.Builder.ServerStateDir, "www", "latest_version"), []byte(b.MixVer), 0644)
```

Comment:
for me it makes more sense to write that file to www/version/ because all formats and latest from formats are in that folder

Reply:
Changed to write the file to www/version/ directory. @otaviobp  Could you take another look?

Reply:
it's better, but we need to address the issue raised by @rchiossi